Adds setup gitlab-webhook command (#728)
tektoncd/pipelines-as-code: docs/content/docs/guide/cli.md

required secrets.
{{< /details >}}

{{< details "tkn pac setup gitlab-webhook" >}}

### Setup GitLab Webhook

> `tkn-pac setup gitlab-webhook`: will allow you to set up a GitLab webhook with pipelines

Reviewer: ```suggestion
`tkn-pac setup gitlab-webhook`: will let you set up a GitLab webhook to interact with Pipelines as Code
```

(you can remove the with pipelines-as-code service)

as code service.

> After setting up webhook, it will provide an option to create Repository and configure it with

Reviewer: ```suggestion
It will let you provide an option to create a Repository and configure it with the
```
